config/environments/development.rb

```diff
@@ -1,7 +1,7 @@
 C2::Application.configure do
   config.action_mailer.preview_path = "#{Rails.root}/lib/mail_previews"
   config.action_mailer.register_preview_interceptor :css_inline_styler
-  config.action_mailer.asset_host = "http://localhost:5000"
+  config.action_mailer.asset_host = "http://2c429f18.ngrok.io" || "http://localhost:5000"
```

Comment:
I *think* that I originally set this because assets were not rendering properly locally. That being said, if it works properly  without I am all for that!